File: 1768-merge-strings-alternately/1768-merge-strings-alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        StringBuilder str = new StringBuilder();
        int i=0;
        while(i<word1.length() || i<word2.length()){
            if(i<word1.length()){
                str.append(word1.charAt(i));
            }if(i<word2.length()){
                str.append(word2.charAt(i));
            }
            i++;
        }
        return str.toString();
    }
}

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.js
/**
 * @param {string} word1
 * @param {string} word2
 * @return {string}
 */
var mergeAlternately = function(word1, word2) {
    let str = "";
    let i = 0,j=0;
    while(i<word1.length && j<word2.length){
        str+=word1.charAt(i);
        str+=word2.charAt(j);
        i++;
        j++;
    }
    while(i<word1.length){
        str+=word1.charAt(i);
        i++;
    }
    while(j<word2.length){
        str+=word2.charAt(j);
        j++;
    }
    return str;
};